type_templates/html.js

```diff
@@ -170,9 +170,9 @@ export default e => {
   let staticPhysicsIds = [];
   {
     const physicsId = physics.addBoxGeometry(
-      new THREE.Vector3(),
-      new THREE.Quaternion(),
-      new THREE.Vector3(width * scale * app.scale.x / (app.scale.x * 2), height * scale * app.scale.y / (app.scale.y * 2), 0.001),
+      object.position,

```

Comment:
Does this need to be `matrixWorld` for nesting support?

Reply:
Updated